src/makeRobotAccountant.js

```diff
@@ -7,6 +7,19 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 0;
+
+  const getSum = (a) => {
+    count++;
+
+    const sum = (b) =>
```

Comment:
The `sum` function is unnecessary. Remove it and directly return the result of the ternary expression inside the `getSum` function.
`return (b) => ....`